floss/render/default.py

```diff
@@ -208,14 +208,14 @@ def render_heading(heading, n, ostream, disable_headers):
     """
     example::
 
-        -------------------------------
+        +-----------------------------+
         | FLOSS STATIC STRINGS (1337) |
-        -------------------------------
+        +-----------------------------+
     """
     if disable_headers:
         return
-    heading = f"| {heading} ({n}) |"
-    ostream.write(tabulate.tabulate([[heading]]))
+    heading = f"  {heading} ({n})  "
```

Comment:
How does one space vs. two spaces look like. Which one do you find nicer?

Reply:
I think one space might look better

Reply:
I think so. Can you update and paste an example output of a run here, please?

Reply:
Hey, I have tried varying the spaces in the code, but I seem to be getting the same output every time. Can you please guide me with this

Reply:
So, I removed the spaces completely does that look better?